From c40aebcb7b1bf0012b63180733909b674f31bfd1 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 29 Apr 2018 23:27:03 +0200
Subject: [PATCH 1/1] efi_loader: if EFI_LOADER is enabled, enable
 CFB_CONSOLE_ANSI

EFI applications like GRUB issue ANSI sequences for setting colors
and for positioning. So if the EFI subsystem is enabled, we should
also enable ANSI control sequences by default.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/efi_loader/Kconfig b/lib/efi_loader/Kconfig
index d38780b604e..c3c4783f394 100644
--- a/lib/efi_loader/Kconfig
+++ b/lib/efi_loader/Kconfig
@@ -10,6 +10,7 @@ config EFI_LOADER
 	default y
 	select LIB_UUID
 	select HAVE_BLOCK_DEVICE
+	imply CFB_CONSOLE_ANSI
 	help
 	  Select this option if you want to run EFI applications (like grub2)
 	  on top of U-Boot. If this option is enabled, U-Boot will expose EFI
-- 
2.17.0

